packs/spell-effects/spell-effect-evolution-surge.json

```diff
@@ -4,7 +4,7 @@
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Evolution Surge]</p>\n<p>Choose one of the following effects:</p><ul><li>Your eidolon gains low-light vision and darkvision.</li><li>You gain @UUID[Compendium.pf2e.bestiary-ability-glossary-srd.Item.Scent] as an imprecise sense up to 30 feet.</li><li>You can breathe underwater and gain a swim Speed equal to your land Speed or 30 feet, whichever is less. If you are aquatic, you can breathe air and gain a land Speed equal your swim Speed or 30 feet, whichever is less. You gain the amphibious trait.</li><li>You gain a +20-foot status bonus to your Speed.</li></ul><hr /><p><strong>Heightened (3rd)</strong></p><ul><li>You become Large, instead of your previous size. This increases your reach to 10 feet but doesn't change any other statistics.</li><li>You gain a climb Speed equal to your land Speed.</li></ul><hr /><p><strong>Heightened (5th)</strong></p><ul><li>Your become Huge, instead of your previous size. This has the same effects as the 3rd-rank option to become Large, except your reach increases to 15 feet.</li><li>You gain a fly Speed equal to its Speed.</li></ul>"
```

Comment:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Evolution Surge]</p>\n<p>Choose one of the following effects:</p><ul><li>You gain low-light vision and darkvision.</li><li>You gain scent as an imprecise sense up to 30 feet.</li><li>You can breathe underwater and gain a swim Speed equal to your land Speed or 30 feet, whichever is less. If you are aquatic, you can breathe air and gain a land Speed equal your swim Speed or 30 feet, whichever is less. You gain the amphibious trait.</li><li>You gain a +20-foot status bonus to your Speed.</li></ul><hr /><p><strong>Heightened (3rd)</strong></p><ul><li>You become Large, instead of your previous size. This increases your reach to 10 feet but doesn't change any other statistics.</li><li>You gain a climb Speed equal to your land Speed.</li></ul><hr /><p><strong>Heightened (5th)</strong></p><ul><li>Your become Huge, instead of your previous size. This has the same effects as the 3rd-rank option to become Large, except your reach increases to 15 feet.</li><li>You gain a fly Speed equal to its Speed.</li></ul>"
```